frontend/app/components/text-component/font-size.js

```diff
@@ -20,21 +23,41 @@ export default Component.extend({
   },
 
   click() {
-    if (this.fonttypeall) {
+    if (this.EmberObject.$('[name="fonttypeall"]')[0].checked === true) {
```

Comment:
instead of using it like this can you bind the object that you want to access in the component, please?
once done, please ask for a review again.